constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintHelper.java

```diff
@@ -112,6 +138,12 @@ protected void init(AttributeSet attrs) {
                 } else if (attr == R.styleable.ConstraintLayout_Layout_constraint_referenced_tags) {
                     mReferenceTags = a.getString(attr);
                     setReferenceTags(mReferenceTags);
+                } else if (attr == R.styleable.ConstraintLayout_Layout_circularflow_angles) {
```

Comment:
Similar -- you should add that in CircularFlow

Reply:
Done ✅